core/http/src/main/java/org/trellisldp/http/LdpResource.java

```diff
@@ -128,6 +139,37 @@ public LdpResource(final ServiceBundler trellis, final String baseUrl) {
         this.trellis = trellis;
     }
 
+    /**
+     * Initialize the Trellis backend.
+     */
+    @PostConstruct
+    public void initialize() {
```

Comment:
In a non-CDI context, how does this get called? Does the responsibility lie with the backend?

Reply:
I assume that it would be up to the wiring system. So, in the Dropwizard wiring, I'd just call `initialize()` right in the code.

Reply:
Okay, can we get an `@apiNote` to that effect, or other doc? I'm a little worried about folks not realizing that they may or may not need to take action depending on their choice of wiring toolbox.

Reply:
Ah yes, definitely